### PR TITLE
Create routes & components for catalog dashboard

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**
Add c (connected) and o (organization) for the routes.
Add routes for catalog dashboard
Created components for catalog dashboard
> Note: routes c & o might change in the future.

### CODE REVIEW

File: apps/catalog/catalog/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -2,19 +2,32 @@ import { NgModule } from '@angular/core';
...
-    path: 'marketplace',
-    loadChildren: () => import('./marketplace/marketplace.module').then(m => m.MarketplaceModule)
+    path: 'auth',
+    loadChildren: () => import('@blockframes/auth').then(m => m.AuthModule)
   },
   {
-    path: 'dashboard',
-    loadChildren: () => import('./dashboard/dashboard.module').then(m => m.DashboardModule)
-  }
+    path: 'c',
```
Comment (Maxstgt) [inline apps/catalog/catalog/src/app/app-routing.module.ts]: no guards?

File: apps/catalog/catalog/src/app/dashboard/dashboard.module.ts [hunk 2]
```diff
@@ -12,13 +12,41 @@ const routes: Routes = [
...
     children: [
       {
-        path: 'overview'
+        path: '',   // Home (dashboard if film, welcome if not)
+        loadChildren: () => import('./pages/home/home.module').then(m => m.HomeModule)
       },
       {
-        path: 'titles'
+        path: 'activity',   // List of notifications
+        // loadChildren: () => import('@blockframes/notifications/pages/list/list.module').then(m => m.NotificationsListModule)
```
Comment (Maxstgt) [inline apps/catalog/catalog/src/app/dashboard/dashboard.module.ts]: mabye a todo if it is not ready?

File: apps/catalog/catalog/src/app/dashboard/title/sales/sales.module.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+@NgModule({
+  declarations: [TitleSalesComponent],
+  imports: [
+    CommonModule,
+    RouterModule.forChild([{ path: '', component: TitleSalesComponent }])
+  ]
+})
+export class TitleSalesModule {
+  constructor() {
+    console.log('CREATED')
```
Comment (Maxstgt) [inline apps/catalog/catalog/src/app/dashboard/title/sales/sales.module.ts]: remove console.log

File: apps/catalog/catalog/src/app/dashboard/title/view/view.module.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+import { RouterModule } from '@angular/router';
+
+import { TitleViewComponent } from './view.component';
+
+const routes = [{
+  path: '',
+  component: TitleViewComponent,
+  children: [
+    {
+      path: '',
```
Comment (Maxstgt) [inline apps/catalog/catalog/src/app/dashboard/title/view/view.module.ts]: why not directly go to sales? not sure if necessary for child routes

File: apps/catalog/catalog/src/app/app-routing.module.ts (hunk 1 above)
Comment (GrandSchtroumpf) [reply apps/catalog/catalog/src/app/app-routing.module.ts]: not yet. Let's focus on building the components first.

File: apps/catalog/catalog/src/app/dashboard/title/view/view.module.ts (hunk 4 above)
Comment (GrandSchtroumpf) [reply apps/catalog/catalog/src/app/dashboard/title/view/view.module.ts]: It's going to be more readable in the routerLink to do : `routerLink="../sales"` than `routerLink=".."`

**Pull request merged**